aip/0153.md

```diff
@@ -0,0 +1,164 @@
+---
+aip:
+  id: 153
+  state: reviewing
+  created: 2019-12-16
+permalink: /153
+redirect_from:
+  - /0153
+---
+
+# Import and export
+
+Many users want to be able to load data into an API, or get their existing data
+out of an API. This is particularly important for enterprise users who are
+concerned about vendor lock.
+
+## Guidance
+
+APIs **may** support import and export when they desire to do so. Additionally,
```

Comment:
I'd remove "when they desire to do so" as redundant - something like:

```text
APIs **may** support import and export operations. These operations **may** create multiple new resources,
or they **may** populate data within a single resource.
```

Reply:
Done this part.